Source/Scene/ClassificationModel.js

```diff
@@ -958,10 +958,11 @@ define([
             return;
         }
 
-        var supportsWebP = FeatureDetection.supportsWebPSync();
-        if (!defined(supportsWebP)) {
+        if (!FeatureDetection.supportsWebP.initialized) {
```

Comment:
I think this whole initialized thing is less clear, why did you get rid of `supportsWebPSync`?